lang/py/avro/schema.py

```diff
@@ -963,6 +1044,25 @@ def to_json(self, names=None):
         to_dump['fields'] = [f.to_json(names) for f in self.fields]
         return to_dump
 
+    def to_canonical_json(self, names):
```

Comment:
…

Reply:
Addressed in eacfdd6a0cd6de9b4e15701d08d9e994f09be310